pgjdbc/src/test/java/org/postgresql/test/jdbc2/CallableStmtTest.java

```diff
@@ -188,6 +216,84 @@ public void testGetNumericWithoutArg() throws Throwable {
   }
 
   @Test
+  public void testGetTimestampWithoutTimeZoneWithoutArg() throws SQLException {
+    assumeCallableStatementsSupported();
+    CallableStatement call = con.prepareCall(func + pkgName + "getTimestampWithoutTimeZoneWithoutArg () }");
+    try {
+      call.registerOutParameter(1, Types.TIMESTAMP);
+      call.execute();
+      assertEquals(Timestamp.valueOf("2004-10-19 10:23:54.000123"), call.getTimestamp(1));
+    } finally {
+      call.close();
+    }
+  }
+
+  @Test
+  public void testGetTimestampWithoutTimeZoneWithoutArgCalendar() throws SQLException {
+    assumeCallableStatementsSupported();
+    CallableStatement call = con.prepareCall(func + pkgName + "getTimestampWithoutTimeZoneWithoutArg () }");
+    try {
+      call.registerOutParameter(1, Types.TIMESTAMP);
+      call.execute();
+      assertEquals(Timestamp.valueOf("2004-10-19 10:23:54.000123"), call.getTimestamp(1, null));
```

Comment:
I tried to make the new tests fit with the style of the existing tests. I don't see an easy, obvious way to parameterize them. The parameters would have to be
- the function name
- the JDBC type
- a lambda to get the actual value
- the expected value

Even then the tests that bind a parameter like `testGetInt` would have to be moved to a different test class.

Let me know if you would like to see them parameterized and I will parameterize them.